static/customview.js

```diff
@@ -110,6 +121,9 @@ function updateMCHCombinedBoxPlot(mmu_gid, hsa_gid) {
 function updateGeneElements() {
     var geneSelected = $('#geneName option:selected').val();
     if (geneSelected != 'Select..') {
+        if (typeof(Storage) !== 'undefined') {
+            localStorage.setItem(species, $('#geneName option:selected').text());
```

Comment:
The storage key is set to be the species the user is currently viewing, so a gene saved as 'mmu' will not be shown if the user is viewing in 'hsa'.